src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -0,0 +1,57 @@
+package frc.robot.subsystems;
+
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
+import com.ctre.phoenix6.hardware.Pigeon2;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class DriveSubsystem extends SubsystemBase {
+    private WPI_TalonSRX leftFrontMotor;
+    private WPI_VictorSPX rightFrontMotor;
+    private WPI_VictorSPX leftBackMotor;
+    private WPI_TalonSRX rightBackMotor;
+    private Pigeon2 pigeon2;
+    public DriveSubsystem() {
+        leftBackMotor = new WPI_VictorSPX(RobotMap.LEFT_BACK_MOTOR_ID);
+        rightBackMotor = new WPI_TalonSRX(RobotMap.RIGHT_BACK_MOTOR_ID);
+        leftFrontMotor = new WPI_TalonSRX(RobotMap.LEFT_FRONT_MOTOR_ID);
+        rightFrontMotor = new WPI_VictorSPX(RobotMap.RIGHT_FRONT_MOTOR_ID);
+        leftFrontMotor.setInverted(true);
+        leftBackMotor.setInverted(true);
+
+
+        pigeon2 = new Pigeon2(RobotMap.PIGEON_ID);
+    }
+    public double getHowMuchLeftMoved(){
+        return leftFrontMotor.getSelectedSensorPosition() / RobotMap.TALON_ENCODER_PPR * RobotMap.WHEEL_RADIUS;
+    }
+    public double getHowMuchRightMoved(){
+        return rightFrontMotor.getSelectedSensorPosition() / RobotMap.TALON_ENCODER_PPR * RobotMap.WHEEL_RADIUS;
+    }
+    public void initialize(){
+        pigeon2.reset();
+        leftFrontMotor.setSelectedSensorPosition(0);
+        rightBackMotor.setSelectedSensorPosition(0);//inits
+    }
+    public double getPigeonAngle(){
```

Comment:
```suggestion
    public double getAngleDegrees(){
```

Pigeon doesn't need to be mentioned. You also want to specific measurement unit.